packages-ts/gauntlet-terra-contracts/networks/.env.mainnet

```diff
@@ -0,0 +1,12 @@
+# From https://www.notion.so/chainlink/Solana-Terra-Multisig-Mainnet-Testing-6cc5a76d99484f778622a041f86b4ec8
```

Comment:
Please remove this file for now, as in rebase it out of history.

Reply:
Do you mean completely remove `.env.mainnet`?
I think it's the one ops guys are actually using.

Reply:
Yes, please remove it completely, and let's sync offline.